WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -236,14 +236,6 @@ character_event = {
 	
 	trigger = {
 		NOT = { year = 800 } # The migration period ended around 800
-		OR = { # Possible target religion groups
-			religion_group = christian
-			religion = finnish_pagan
-			religion = germanic_pagan
-			religion = baltic_pagan
-			religion = scythian_pagan
-			religion = slavic_pagan
-		}
```

Comment:
Ah, I was afraid it was a restriction on the player.

I still don't see why we should limit the AI in their migration. They chose to head west in RL, but nothing was stopping them from going east or south. Having them migrate unpredictably would be more realistic than saying where they can migrate.

I'd prefer a means in which they are _more likely_ to migrate west, and if they are Lunatics, they should have a more likely chance _not_ to migrate west. THis way, there is always a chance to go to some bizarre place, but they are most likely to follow the historic route.